README.md

```diff
@@ -1,59 +1,207 @@
+
+![truffle compile](/images/image-02.png)
+
+```shell
+migrate
+```
+
+And the `migrate output` should be similar to:
+
+![truffle migrate](/images/image-03.png)
+
+6. Running contract tests.
+
+Our box also comes with the file `TestSimpleStorage.js` for testing the smart contract. You can check it out in the `test` folder.
+
+Run this command in the development console:
+
+```shell
+test
+```
+
+This `test output` should be similar to:
+
+![truffle test](/images/image-04.png)
+
+Note the command varies slightly if you're in or outside of the development console.
+
+```javascript
+// inside the development console.
+test
+
+// outside the development console.
+truffle test
+```
+
+## The application
+
+Our box is a client-server: 
+- Server uses Express JS to interact with the smart contract.
+- Client uses React app.
+
+7. Running in developer mode
+
+In another terminal (i.e. not in the truffle develop prompt), go to the `app` directory and run the app in developer mode.
+
+> Do not close the other terminal, which is running the Truffle development console, because it is our Blockchain simulator.
+> 
+> If you close it and then open it again, you need to deploy / migrate the smart contract again too!
+
+```shell
+cd app
+npm run dev
+```
+
+This command executes server and client applications and you can access in your browser:
+
+- Server: [http://localhost:8080/](http://localhost:8080/)
+- Client: [http://localhost:3000/](http://localhost:3000/)
+
+> Smart contract changes must be manually recompiled and migrated!
+
+**NOTE**: This box is the starting point for the RSK tutorial [using rsk-react-express-box](https://developers.rsk.co/tutorials/truffle-boxes/rsk-react-express-box/).
+
+8. Running the production server
+
+You can chhose to run only the production server. In another terminal (i.e. not in the truffle develop prompt), go to the `app` directory and run the app in production mode.
+
+```shell
+cd app
+npm start
+```
+
+> If you have `yarn` installed, you can use `yarn start`.
+
+Then go to your browser at [http://localhost:8080/](http://localhost:8080/)
+
+![localhost port 8080](/images/image-11.png)
+
+> When you are running only the production server, for any change you need to stop and run the server again for the updates to take effect!
+
+9. To build the application for production, use the build script in the `app` folder. A production build will be in the `app/dist` folder.
+
+```javascript
+npm run build
+```
+
+Take a look in the `app/dist` folder:
+
+![dist folder](/images/image-13.png)
+
+## Using RSK networks
 
 ### Setup an account & get R-BTC
 
-- Get an address using [these instructions](https://developers.rsk.co/rsk/architecture/account-based/ "Account Based RSK Addresses - RSK Developers Portal").
+- Get an address, learning how works the [account based RSK addresses](https://developers.rsk.co/rsk/architecture/account-based/ "Account based RSK addresses - RSK Developers Portal").
```

Comment:
```suggestion
- Get an address, using the [Account based RSK addresses](https://developers.rsk.co/rsk/architecture/account-based/ "Account based RSK addresses - RSK Developers Portal") to learn how it works.
```

Reply:
No, the page [account-based](https://developers.rsk.co/rsk/architecture/account-based/) didn't teach how to get an address. After you get an address (and I am not explain how to do it here), this page explains how the addresses work in RSK.